index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+<!-- Required meta tags -->
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+<!-- Bootstrap CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+<link  rel="stylesheet"href=C:\Users\User\OneDrive\Desktop\tracking_the_blue\stylesheet.css>
+
+    <title>Tracking The Blue</title>
+
+  </head>
+  <body>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+
+
+    <div id =content class="d-flex justify-content-center" >
+ <!--TODO: need to fix Jumbotron so that it fills the screen, jmbotron-fluid should
+        Have fixed this, but it fills it's parrent element, but it's parrent item doese not fill the screen-->
+          
+    <div class="container-fluid">
+        <div class="jumbotron jumbotron-fluid" style="background-color: blue; color: yellow;">
+        <div>
+            <h1>Tracking the Blue</h1>
+        </div>
+
+        </div>
+    
+        <div id="about_the_project">
+            <p>Isum Copy... sdfskdhf;kjhsdjfh</p>    
+                <p>sdfjhk sdjfhksdhf</p> 
+                <p>skdkfjlsdkjf sdfkjsdl sdlfkj dirn sldkfj</p> 
+                <p>KDSjflskjd  twefkljs kjliji nfdsnl. </p>
+            </p>

```

Comment:
We'll want to replace this with actual text. We don't want placeholder text like this on the website even if it's not fully polished yet.

Reply:
Next time you are on the stream maybe you and @AndyM84 can talk about this one.

Reply:
Well, we don't have differing points of view, as this is no longer a mockup 😉  Now that we're building it, yeah, you should ask us for the content and then plop it in.

For clarity, @kwhite17, I had said while we were doing mockups that they should avoid providing mockups with real copy in them since it has the chance to distract the discussion to the wrong things.

Reply:
That makes sense now! I'm learning so much!

Reply:
@AndyM84  and @kwhite17  What would you like me to put for the copy? ;)

Reply:
For now, just put the title "Track the Blue" and then some lorem ipsum from something like https://www.loremipzum.com/en/